src/Parquet/Reader.hs

```diff
@@ -1,4 +1,6 @@
 {-# LANGUAGE DataKinds #-}
+{-# LANGUAGE ViewPatterns #-}
```

Comment:
We should probably add a list of default extensions we like in `package.yaml` that's known to interact well. I can do it in a later MR or feel free to do it here.

Reply:
I've taken care of this in https://github.com/yigitozkavci/parquet-hs/pull/10/.